### PR TITLE
long running arc with a CapturePerson particle rejoining existing arc

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: particles/PipeApps/AndroidAutofill.arcs [hunk 1]
```diff
@@ -31,14 +31,24 @@ recipe AndroidAutofill
...
+//     start
+//     [{
+//       "firstName": "John",
+//       "lastName": "Smith",
+//       "password": "qwerty123",
+//       "phone": "987-6543-210",
+//       "postalAddress": "123 Main st."
+//     }]
+
+// store RecentPeople of [Person] #recentPeople in PeopleJson
```
Comment (csilvestrini) [inline particles/PipeApps/AndroidAutofill.arcs]: Just delete this?

File: src/javaharness/java/arcs/android/client/RemotePec.java [hunk 2]
```diff
@@ -58,21 +75,27 @@ public void runArc(String recipe, Particle particle) {
...
+    if (getArcId() == null) {
+      setArcId(Id.newArcId().toString());
+    }
+
+    if (pecId == null) {
+      setPecId(idGenerator.newChildId(Id.fromString(arcId), "pec").toString());
+    }
+
+    if (particle.getId() == null) {
+      particle.setId(idGenerator.newChildId(Id.fromString(arcId), "particle").toString()); //particleId.toString());
```
Comment (csilvestrini) [inline src/javaharness/java/arcs/android/client/RemotePec.java]: Delete comment

File: particles/PipeApps/AndroidAutofill.arcs (hunk 1 above)
Comment (mmandlis) [reply particles/PipeApps/AndroidAutofill.arcs]: done

File: src/javaharness/java/arcs/android/client/RemotePec.java (hunk 2 above)
Comment (mmandlis) [reply src/javaharness/java/arcs/android/client/RemotePec.java]: done

**Pull request merged**